### PR TITLE
MULE-19717: Add serializer stereotype

### PR DESCRIPTION
**Pull request by marianorean**
(cherry picked from commit 9a16b7e2cd6035cd9d37068dc78c2e1545775d14)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -30,7 +30,7 @@
 
   <properties>
     <muleSchedulerServiceTestVersion>1.3.3-SNAPSHOT</muleSchedulerServiceTestVersion>
-    <muleHttpServiceTestVersion>1.5.14-SNAPSHOT</muleHttpServiceTestVersion>
+    <muleHttpServiceTestVersion>1.5.15-SNAPSHOT</muleHttpServiceTestVersion>
```
Comment (elrodro83) [inline pom.xml:33]: this is unrelated to this pr

**Review by elrodro83 — CHANGES_REQUESTED** [review]

**Review by elrodro83 — APPROVED** [review]

**Review by pabloperalta — APPROVED** [review]
